### DISCUSSION TITLE
LPS question

### DISCUSSION THREAD

**Discussion by CalvinKoung** [bitcraze/discussions#1143] [Q&A]
Hi, If I start the equippment will all anchors be time synchronized ? tks

**Comment by knmcguire** [comment]
With time synchronized, do you mean that the clocks on each chip of the loco node have adjusted their clock to be the same on each node?

I don't think that happens but each UWB packet does include a timestamp from the other anchors... so I think something happens with that instead (but need to go deeper into this). Here is the documentation for this that might point you in the right path: https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/protocols/

**Reply by CalvinKoung** [reply]
Thanks your response, can you give me the document address again ?

**Reply by knmcguire** [reply]
Ah woops I've updated it. Here is the link to protocol packages description: https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/protocols/

**Reply by CalvinKoung** [reply]
I couldn't found answer. So the LPS anchor can't set the one anchor is master and others are slave to sync time to master ? tks

**Reply by knmcguire** [reply]
I don't think that works like that. It is all done by keeping in mind the relative time of those timestamps. So no there is no master clock it seems

**Reply by CalvinKoung** [reply]
But if no internal clock sync, we don't need to compensate the time to reduce error ?

**Reply by knmcguire** [reply]
It is not that the clock are synced, but that the drift in timing is monitored, and that is why in all the packing there are so many timestamps included as one of the reasons. I believe perhaps that this documentation also explains a bit about that. https://www.bitcraze.io/documentation/repository/lps-node-firmware/master/functional-areas/tdoa3_implementation/. It has more to do that at one point the packages will collide with eachother, but for the ranging itself only 'relative timing' is necessary
